lib/netconf/static.go

```diff
@@ -118,7 +118,7 @@ func (s Static) ExecutionSeeds() []string {
 
 func (s Static) ExecutionRPC() string {
 	if s.Network == Devnet {
-		return "http://localhost:8001"
+		return "http://127.0.0.1:8000"
```

Comment:
if we don't want to panic, at least return invalid URL so that it is clear that this should not be used and will error. I suggest: `no well known ephemeral rpc`